source/images/burst1.svg

```diff
@@ -1,4 +1,4 @@
-<svg width="178" height="157" viewBox="0 0 178 157" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
@gvn is `viewBox` safe to remove? 👀

Reply:
I'm not sure...I don't usually hand-optimize SVG files.